auth/openid_connect.md

```diff
@@ -14,10 +14,8 @@ been tested with KeyCloak but is implemented generically using Apache’s
 mod\_auth\_openidc module and should work with other OIDC Identity
 Providers.
 
-The current implementation only secures the Appliance’s Web
-administrative UI with OIDC.
-
-**Note:** The Self-Service UI does not currently support OIDC.
+The current implementation secures the Appliance’s Web
+administrative and self service UI with OIDC.
```

Comment:
@abellotti I think `self service` should be `Self-Service`. So this should read:
`administrative and Self-Service UI with OIDC.`